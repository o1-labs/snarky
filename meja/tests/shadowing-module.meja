module M = {
  type t;
};

module M = {
  type u;
};
