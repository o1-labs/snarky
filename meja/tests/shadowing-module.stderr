File "tests/shadowing-module.meja", line 5, characters 7-8:
Error: There is already a module with name M.
